x-pack/plugins/endpoint/server/plugin.ts

```diff
@@ -34,9 +36,16 @@ export class EndpointPlugin
       EndpointPluginStartDependencies
     > {
   private readonly logger: Logger;
+  private readonly endpointAppContextService: EndpointAppContextService;
   constructor(private readonly initializerContext: PluginInitializerContext) {
     this.logger = this.initializerContext.logger.get('endpoint');
+    this.endpointAppContextService = new EndpointAppContextService();
+  }
+
+  public getEndpointAppContextService(): EndpointAppContextService {
```

Comment:
Bummer that we need to expose this just for tests. I can't think of a better way to get around it though.